hcp/Operator_Workload_Scheduling_Issue.json

```diff
@@ -0,0 +1,9 @@
+{
+      "severity": "Major",
+      "service_name": "SREManualAction",
+      "log_type": "cluster-lifecycle",
+      "summary": "Operator workload scheduling issue requires action",
+      "description": "Your cluster's ${OPERATOR_WORKLOAD} cannot be scheduled due to ${REASON}. To resolve this issue, please ensure your cluster has adequate worker node capacity to support the desired number of replicas.",
+      "doc_references": ["https://docs.redhat.com/en/documentation/openshift_container_platform/4.19/html/machine_management/manually-scaling-machineset"],
```

Comment:
These are OpenShift docs, not specific to ROSA. Can we find docs for doing this on ROSA for consistency?